src/v0/util/index.js

```diff
@@ -1816,7 +1816,7 @@ const simpleProcessRouterDest = async (inputs, singleTfFunc, reqMetadata, proces
         let resp = input.message;
         // transform if not already done
         if (!input.message.statusCode) {
-          resp = await singleTfFunc(input, processParams);
+          resp = await singleTfFunc(input, processParams, reqMetadata);
```

Comment:
Any impact in production due to reqMetadata not being passed till now ?
I see one use-case depending on this https://github.com/rudderlabs/rudder-transformer/blob/feat.braze-alias-batching/src/v0/destinations/braze/transform.js#L385